app/assets/javascripts/map.js

```diff
@@ -28,17 +28,19 @@
         markers = L.layerGroup();
       }
       marker = null;
-      markerIcon = L.divIcon({
-        className: "map-marker",
-        iconSize: [30, 30],
-        iconAnchor: [15, 40],
-        html: '<div class="map-icon"></div>'
-      });
-      createMarker = function(latitude, longitude) {
+      markerIcon = function(alt_text) {
+        return L.divIcon({
+          className: "map-marker",
+          iconSize: [30, 30],
+          iconAnchor: [15, 40],
+          html: '<div class="map-icon" aria-label="' + alt_text + '"></div>'
+        });
+      };
+      createMarker = function(latitude, longitude, text) {
```

Comment:
I noticed `createMarker` now takes a third `text` argument, but in line 65 we still call it like this:

```  
marker = createMarker(e.latlng.lat, e.latlng.lng);
```

This seems to end up rendering an `aria-label="undefined"`.